openshift_tools/monitoring/zagg_sender.py

```diff
@@ -135,7 +131,7 @@ def add_heartbeat(self, heartbeat, host=None):
                                                  )
         self.unique_metrics.append(hb_metric)
 
-    def add_zabbix_keys(self, zabbix_keys, host=None, synthetic=False):
+    def _add_zabbix_keys(self, zabbix_keys, host=None, synthetic=False):
```

Comment:
To be clear, it's not backwards compatible because of the _ at the front.  Backwards compatible would be to leave the name exactly the same.

Can you explain the change tracking part in more detail?

We use git for change tracking, so I'm not sure why this is needed.